en-US/Players_Guide/04_Adventuring/Combat_Movement_and_Position.md

```diff
@@ -0,0 +1,132 @@
+
+Your movement can include jumping, climbing, and swimming.
+These different modes of movement can be combined with walking, or they can constitute your entire move.
+However you’re moving, you deduct the distance of each part of your move from your speed until it is used up or until you are done moving.
+
+##### Breaking Up Your Move
+
+You can break up your movement on your turn, using some of your speed before and after your action.
+For example, if you have a speed of 30 feet, you can move 10 feet, take your action, and then move 20 feet.
+
+###### Moving between Attacks
+
+If you take an action that includes more than one weapon attack, you can break up your movement even further by moving between those attacks.
+For example, a Fighter who can make two attacks with the Extra Attack feature and who has a speed of 25 feet could move 10 feet, make an attack, move 15 feet, and then attack again.
+
+###### Using Different Speeds
+
+If you have more than one speed, such as your walking speed and a flying speed, you can switch back and forth between your speeds during your move.
+Whenever you switch, subtract the distance you’ve already moved from the new speed.
+The result determines how much farther you can move.
+If the result is 0 or less, you can’t use the new speed during the current move.
+
+For example, if you have a speed of 30 and a flying speed of 60 because a Wizard cast the Fly spell on you, you could fly 20 feet, then walk 10 feet, and then leap into the air to fly 30 feet more.
+
+##### Difficult Terrain
+
+Combat rarely takes place in bare rooms or on featureless plains.
+Boulder-strewn caverns, briar* choked forests, treacherous staircases—the setting of a typical fight contains difficult terrain.
+
+Every foot of movement in difficult terrain costs 1 extra foot.
+This rule is true even if multiple things in a space count as difficult terrain.
+
+Low furniture, rubble, undergrowth, steep stairs, snow, and shallow bogs are examples of difficult terrain.
+The space of another creature, whether hostile or not, also counts as difficult terrain.
+
+##### Being Prone
+
+Combatants often find themselves lying on the ground, either because they are knocked down or because they throw themselves down.
+In the game, they are prone, a condition described in [chapter <mark>XX</mark>, "Conditions"](#Conditions_conditions).
+
+You can drop prone without using any of your speed.
+Standing up takes more effort; doing so costs an amount of movement equal to half your speed.
+For example, if your speed is 30 feet, you must spend 15 feet of movement to stand up.
+You can’t stand up if you don’t have enough movement left or if your speed is 0.
+
+To move while prone, you must crawl or use magic such as teleportation.
+Every foot of movement while crawling costs 1 extra foot.
+Crawling 1 foot in difficult terrain, therefore, costs 3 feet of movement.
+
+> **Interacting with Objects Around You** {#Combat_Movement_and_Position_interacting_with_objects}
```

Comment:
This is a block / sidebar in the WotC SRD, so I added it as such here as well. (We can change it if requested of course.)